prediction-applier-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-prediction-applier-extension/alfresco-global.properties

```diff
@@ -3,3 +3,11 @@
 ## System Administrators can override these values in environment specific configurations in
 ## alfresco/tomcat/shared/classes/alfresco-global.properties.
 ##
+auth.providers.hyland-experience.type=oauth2
```

Comment:
I suggest we start all our properties with `hxi.` to avoid collisions.

Reply:
done!